docs/starters.yml

```
@@ -1634,3 +1634,20 @@
     - GSAP ready
     - Nice devtools
     - Decorators
+- url: https://gatsby-sseon-starter.netlify.com/
+  repo: https://github.com/SeonHyungJo/gatsby-sseon-starter
+  description: Simple starter template for Gatsby
+  tags:
+    - Simple
+    - Blog
+    - Disqus
+    - Portfolio
+    - Markdown
+    - Sseon
```

Sseon is not really a tag we are supporting however `Styling:SCSS` would be a good fit for you starter
```suggestion
    - Styling:SCSS
```